### ISSUE TITLE
Downloading Just Assets from RCloud

### ISSUE THREAD

**Issue by danielra0091** [att/rcloud#2579]
Would it be possible to add a feature to:
1) Download individual asset files
2) Download all assets from one command

![asset](https://user-images.githubusercontent.com/28115584/41607729-49a67b62-73ac-11e8-8bd7-9cddc5a164f1.png)

**Comment by gordonwoodhull** [comment]
That's a lot of assets!!!

You can open the current asset in a new tab by clicking on the "link" link in the lower left hand corner of the asset panel

![link-link](https://user-images.githubusercontent.com/1366709/41617839-930cc978-73cf-11e8-8301-1d94b93abf1c.png)

If you right-click it, you have the ability to save the content to a file. 

As for saving all assets, I'd be more in favor of providing a way to clone the whole repo behind the notebook, rather than adding a button that does that. If we did provide a way to download the assets, maybe just zipping the whole notebook and downloading as a zip file would be safer than starting dozens of downloads.

**Comment by danielra0091** [comment]
Yeah the zipfile option would be great. Anything that helps avoid saving each one individually would be a big help.
